cms/templates/settings.html

```diff
@@ -292,6 +292,26 @@ <h2 class="title-2">${_("Introducing Your Course")}</h2>
                 <span class="tip">${_("Information for prospective students")}</span>
               </header>
               <ol class="list-input">
+
+                % if about_page_editable:
+                <li class="field" id="field-course-title">
+                  <label for="course-title">${_("Course Title")}</label>
+                  <input type="text" id="course-title" data-display-name="${context_course.display_name}">
```

Comment:
This page does not have the proper filter set to protected against XSS. If you set the course display name to have a script attack, you will see the script be executed when the page is loaded.

Part of adding to this page needs to include updating it per http://edx.readthedocs.org/projects/edx-developer-guide/en/latest/conventions/safe_templates.html. I get particularly concerned when I see context_course.display_name above, as that value will not be escaped.